### PR TITLE
document repository settings

### LINKED ISSUE #52: Merging Process
I noticed that several people approved #51 but no one merged it. What should our process for this look like?

1. The one who created the PR is in charge for merging
  I think that's good  for PRs like #1, where it's good that multiple persons have a look. The downside is that it only works if the PR creator has write permission for this repository. As we want outside contributions that option can be excluded as a general process..
1. The one who reviews the PR also merges it
    That's simple and straight forward. Downside is that people have to get back to the PR in case they approve it before status checks have been finished.
1. Implement auto merge based on criteria e.g.
    - PR has at least one approved review
    - all required status checks are passing (chart linting, superlinter, DCO)
    That's similar to what we had in stable repository. We would need to investigate how to implement it. 

For option 3 we could look into https://github.com/pascalgn/automerge-action

Which option would you prefer?

### PR DESCRIPTION
**Pull request by torstenwalter**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #52

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed

### CODE REVIEW

**Review by monotek — APPROVED** [review]

File: PROCESSES.md
Comment (monotek) [inline PROCESSES.md:13]: As suggested in https://github.com/prometheus-community/helm-charts/pull/68#issuecomment-695843546 it might be worth to add a timewindow how long repo admin should wait for the original maintainer to approve.

Imho 2 weeks should be maximum.

**Review by monotek — CHANGES_REQUESTED** [review]

File: PROCESSES.md
Comment (torstenwalter) [reply PROCESSES.md:13]: Do you have a suggestion how to make the two weeks more clear than what is already stated?

Comment (monotek) [reply PROCESSES.md:13]: Woops! I've been looking for a number in the text :D 
Sorry.

File: PROCESSES.md
Comment (gkarthiks) [inline PROCESSES.md:10]: just a comma(,) punctuation between the "it it" like following

```
have a look on it, it should be mentioned as a comment
```

File: PROCESSES.md
Comment (gkarthiks) [inline PROCESSES.md:101]: double typed `branch`

**Comment by torstenwalter** [comment]
@gkarthiks Thank you for the feedback. Both changes are done.

**Review by gkarthiks — DISMISSED** [review]

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md:48]:
```suggestion
This allows them to manage issues, review PRs etc according to the rules in [CODEOWNERS](./.github/CODEOWNERS).
To request adding a user to [@prometheus-community/helm-charts-maintainers](https://github.com/orgs/prometheus-community/teams/helm-charts-maintainers), ask [@prometheus-community/helm-charts-admins](https://github.com/orgs/prometheus-community/teams/helm-charts-admins) in the corresponding issue or pull request.
```

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md]: **Quick note:** reminder to both of us that the `gh-pages` branch is not currently protected against force-pushing, and to double-check up to see if this is still necessary for `helm/chart-releaser` (it used to require force-pushing, but that may have changed and it didn't register with me).

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md:47]: We can't link to teams, but we can adopt the syntax 😄 see https://github.com/github/markup/issues/209
```suggestion
Chart maintainers are members of [@prometheus-community/helm-charts-maintainers](https://github.com/orgs/prometheus-community/teams/helm-charts-maintainers).
```

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md:100]: **Quick note:** reminder to both of us that the `gh-pages` branch is not currently protected against force-pushing, and to double-check up to see if this is still necessary for `helm/chart-releaser` (it used to require force-pushing, but that may have changed and it didn't register with me).

**Review by scottrigby — CHANGES_REQUESTED** [review]
a few suggestions

File: .github/CODEOWNERS
Comment (torstenwalter) [reply .github/CODEOWNERS:9]: It's just to get both aligned. Adding him on the chart requires a version bump anyhow. I could also keep it here as it was before.

Comment (scottrigby) [reply .github/CODEOWNERS:9]: I would keep here for now only so that they can still help review PRs in the meantime. I opened https://github.com/prometheus-community/helm-charts/issues/153 to address this separately.

**Review by monotek — DISMISSED** [review]

File: .github/CODEOWNERS
Comment (torstenwalter) [reply .github/CODEOWNERS]: updated

File: PROCESSES.md
Comment (torstenwalter) [reply PROCESSES.md]: It's protected for https://github.com/jenkinsci/helm-charts and works well. I can check the settings over there and and document them as another PR. You can verify them and after that we can configure those.

Looks like following the process outlined in this document ;-)

File: .github/CODEOWNERS
Comment (scottrigby) [inline .github/CODEOWNERS:13]: One thing I just noticed, the check CODEOWNERS Script should sort maintainer names. Now it's probably sorting based on the order in Chart.yaml, but for readability let's throw a sort in that script. And update this PR too. Do you agree?

**Review by scottrigby — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (torstenwalter) [reply .github/CODEOWNERS]: I had it sorted first and was not sure if we want this or want to keep same order as in `Chart.yaml`.
As you raised the issue now as well let's do the sorting ;-)

**Review by monotek — DISMISSED** [review]

File: .github/CODEOWNERS
Comment (scottrigby) [inline .github/CODEOWNERS:9]: Just this line still needs to be sorted, right?

**Review by scottrigby — APPROVED** [review]
Perfect 👍 thanks for all the fixes

**Pull request merged**
